src/Uno.Extensions.Core/PlatformHelper.cs

```diff
@@ -56,6 +60,47 @@ public static bool IsAppPackaged
 		}
 	}
 
+	internal static void SetAppAssembly(Assembly assembly) => _appAssembly = assembly;
```

Comment:
We should probably make this and `GetAppAssembly()` public, as we may need to use its functionality in e.g. https://github.com/unoplatform/ShowMeTheXAML/blob/aae225c269c7fa267d26738b615c61a769f55399/ShowMeTheXAML/XamlDisplay.cs#L67.

We may also want to make the parameter nullable, so that `PlatformHelper.SetAppAssembly(null)` is permitted.  This would mean "use the default value next time."